src/core/server/services/comments/pipeline/phases/toxic.ts

```diff
@@ -79,7 +79,7 @@ export const toxic: IntermediateModerationPhase = async ({
   }
 
   // TODO: (wyattjoh) replace hardcoded default with config.
-  const timeout = ms("500ms");
+  const timeout = ms("800ms");
```

Comment:
Reason to up to 800ms? API getting slower on average?

Reply:
During my testing there was slowdowns. It is planned to replace this with a config variable, just hasn't been implemented yet.